### PR TITLE
Make NNN_COLORS override NO_COLORS

### PR DESCRIPTION
**Pull request by leovilok**
This is the minimal patch, but we could save a `getenv` call with very few more code.

### CODE REVIEW

File: src/nnn.c [hunk 1]
```diff
@@ -1498,7 +1498,7 @@ static bool initcurses(void *oldmask)
 #endif
 	curs_set(FALSE); /* Hide cursor */
 
-	if (!getenv("NO_COLOR")) {
+	if (!getenv("NO_COLOR") || getenv(env_cfg[NNN_COLORS])) {
```
Comment (jarun) [inline src/nnn.c:1501]: Just assign `colors` above this line and we won't need to `getenv(NNN_COLORS)` twice.

Comment (leovilok) [reply src/nnn.c]: Done.

File: nnn.1 [hunk 2]
```diff
@@ -417,7 +417,7 @@ separated by \fI;\fR:
 \fBnnn:\fR this is a special variable set to the hovered entry before executing
 a command from the command prompt or spawning a shell.
 .Pp
-\fBNO_COLOR:\fR if defined, disable ANSI color output.
+\fBNO_COLOR:\fR if defined, disable ANSI color output (except if NNN_COLORS is defined).
```
Comment (jarun) [inline nnn.1:420]: \fBNO_COLOR:\fR disable ANSI color output (overridden by \fBNNN_COLORS\fR).

File: src/nnn.c [hunk 3]
```diff
@@ -1498,9 +1498,11 @@ static bool initcurses(void *oldmask)
 #endif
 	curs_set(FALSE); /* Hide cursor */
 
-	if (!getenv("NO_COLOR")) {
+	char *colors = getenv(env_cfg[NNN_COLORS]);
```
Comment (jarun) [inline src/nnn.c:1501]: Add newline after declaration (lint complains). Also after line 1503.

File: nnn.1 (hunk 2 above)
Comment (leovilok) [reply nnn.1]: Done.

File: src/nnn.c (hunk 3 above)
Comment (leovilok) [reply src/nnn.c:1501]: Done.

File: nnn.1 [hunk 4]
```diff
@@ -417,7 +417,7 @@ separated by \fI;\fR:
 \fBnnn:\fR this is a special variable set to the hovered entry before executing
 a command from the command prompt or spawning a shell.
 .Pp
-\fBNO_COLOR:\fR if defined, disable ANSI color output.
+\fBNO_COLOR:\fR if defined, disable ANSI color output (overridden by \fBNNN_COLORS\fR).
```
Comment (jarun) [inline nnn.1:420]: Please remove the `if defined,` also. Just keep:

    disable ANSI color output (overridden by \fBNNN_COLORS\fR).

Comment (leovilok) [reply nnn.1]: Done.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**